core/src/main/scala/cats/syntax/unorderedFoldable.scala

```diff
@@ -29,6 +29,23 @@ trait UnorderedFoldableSyntax extends UnorderedFoldable.ToUnorderedFoldableOps {
 
 final class UnorderedFoldableOps[F[_], A](private val fa: F[A]) extends AnyVal {
 
+  /**
+   * test if `F[A]` contains an `A`, named contains_ to avoid conflict with existing contains which uses universal equality
+   *
+   * Example:
+   * {{{
+   * scala> import cats.implicits._
```

Comment:
While we're here ...
```suggestion
   * scala> import cats.syntax.all._
```